diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -54,6 +54,11 @@ fun ASTNode.getAllChildrenWithType(elementType: IElementType): List<ASTNode> =
 fun ASTNode.getFirstChildWithType(elementType: IElementType): ASTNode? =
         this.getChildren(null).find { it.elementType == elementType }
 
+fun ASTNode.isFollowedByNewline() =
+        this.treeNext.elementType == WHITE_SPACE && this.treeNext.text.contains("\n")
+
+fun ASTNode.isBeginByNewline() =
```

Comment:
I suggest to call this method something like `hasNewlineAfter`. Also, these methods are currently identical. Please fix this method and use it in your rule.